webClient/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "2.3.0",
   "license": "EPL-2.0",
   "scripts": {
-    "start": "NODE_OPTIONS=--max_old_space_size=4096 || set NODE_OPTIONS=--max_old_space_size=4096 && webpack --config ./webpack.build.config.js --watch --progress",
-    "build": "NODE_OPTIONS=--max_old_space_size=4096 || set NODE_OPTIONS=--max_old_space_size=4096 && webpack --config ./webpack.build.config.js --progress && node prune.js",
+    "start": "export NODE_OPTIONS=--max_old_space_size=4096 || set NODE_OPTIONS=--max_old_space_size=4096 && webpack --config ./webpack.build.config.js --watch --progress",
```

Comment:
cross-env is good package to use here, it works for different platform

Reply:
https://github.com/zowe/explorer-jes/blob/17f8c9beffb595ff64ec6c7b3a5014e9a38b1491/package.json#L92